### ISSUE TITLE
makeRequest missing http method

### ISSUE THREAD

**Issue by tobyfan1980** [aliyun/aliyun-openapi-cpp-sdk#74]
* **Product and API**:
* **Platform**:
* **How to Reproduce**

the generated code shows that when client class calls makeRequest, it does not set the third parameter HttpRequest::Method, as a result, all requests are using GET. 

following code is an example of a generated POST api 
```
EasClient::CreateServiceOutcome EasClient::createService(const CreateServiceRequest &request) const
{
	auto endpointOutcome = endpointProvider_->getEndpoint();
	if (!endpointOutcome.isSuccess())
		return CreateServiceOutcome(endpointOutcome.error());

	auto outcome = makeRequest(endpointOutcome.result(), request);

	if (outcome.isSuccess())
		return CreateServiceOutcome(CreateServiceResult(outcome.result()));
	else
		return CreateServiceOutcome(outcome.error());
}
```

**Comment by AxiosCros** [comment]
thanks for your feedback.

It will be fixed in the next commit.

**Comment by AxiosCros** [comment]
fixed in $74,#75
